ci-operator/step-registry/insights-live/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- dptp
```

Comment:
huh do we really want to own this?

Reply:
I wasn't sure who is the owner so I added us until I figure it out.

Reply:
Yeah, sounds good